src/SQLQueryStress/Program.cs

```diff
@@ -54,6 +54,7 @@ private static void DisplayHelp<T>(ParserResult<T> result, IEnumerable<Error> er
                     help.AddPreOptionsLine("Check for updates at: https://github.com/ErikEJ/SqlQueryStress");
                     help.AddPreOptionsLine("Sample usage:");
                     help.AddPreOptionsLine("SqlQueryStress -c saved.SqlStress -u -t 32 -d sqldb.perfenv.mycompany.com -r results.csv");

```

Comment:
Should this line not be removed?

Reply:
@ErikEJ, I would assume so. But, prior to removing that line, I will do some testing around path names (full file path vs just file name). I will also try to fix the code to accept both versions.